templates/releases.hbs

```diff
@@ -14,7 +14,7 @@
   <h1 class="brand"><i class="fa fa-cubes fa-fw"></i> Docs.rs</h1>
 
   <form action="/releases/search" method="GET" class="landing-search-form">
-    <div><input class="search-input" id="search" name="query" type="text" placeholder="Click or press 'S' to search" autofocus></div>
+    <div><input class="search-input" id="search" name="query" type="text" aria-label="Find crate by search query" placeholder="Click or press 'S' to search" autofocus></div>
```

Comment:
The sentence here doesn't sound very nice... What about "Input to search crate"?

Reply:
See below. We can nitpick language all day, but it isn't helpful unless critiques align with accessibility specs (which I also linked in the text when opening this issue. Please review before issuing further critique, because it's obvious you're lacking context.)